.github/actions/clp-core-build/action.yaml

```diff
@@ -63,7 +63,7 @@ runs:
 
         mkdir -p "${output_dir}"
         cd "$GITHUB_WORKSPACE/components/core/build"
-        cp clg clp clp-s glt make-dictionaries-readable "${output_dir}"
+        tar cf "${output_dir}/clp.tar" clg clp clp-s glt make-dictionaries-readable
```

Comment:
How about 
`tar cfvv`
?

With Level 2 verbose, from the automation logs we can view the permissions of the files archived.